app/Resources/assets/scss/pages/assistants.scss

```diff
@@ -2,15 +2,9 @@
 
 .assistants {
   section {
-    margin: 125px auto;
```

Comment:
Var det meningen å redusere margin på alle sections, eller skulle det kun være på den rett under den nye CTA-blokken?

Reply:
Det skulle vel egentlig det ja

Reply:
Ja. Resten av innholdet ble dytta litt for langt ned syns jeg

Reply:
Okei, det går sikkert fint å redusere litt. Jeg foreslår 80-100px, og at vi da også reduserer til samme margin på /laerere og /omvektor slik at det blir likt. Hva tenker du, blir det fortsatt for mye?

Reply:
⬆️ Og da tenker jeg på alle sections

Reply:
Kan ta det i en annen PR

Reply:
Nå bare gjorde jeg headeren like stor som før (den pleide å være `page-header-thin`) og satt alle `assistants.sections` til å ha samme margin som den. Eneste som plager meg er at p-taggen i headeren har en liten `margin-bottom` så CTAen kommer ikke midt mellom de to likevel :thinking: 

![image](https://user-images.githubusercontent.com/5937246/44022855-287b184a-9ee9-11e8-9db7-ad34b7385f55.png)

Hvordan syns du det blir?

Reply:
Det går fint. Fjerner margin-bottom på p-taggen i en annen PR

Reply:
Ok, nice

Reply:
Hmm. Tror kanskje CTA-boksen må lengre opp, at den får `margin-top: -60px` eller noe sånt